outfit.js

```diff
@@ -1,9 +1,9 @@
 class Outfit {
-  constructor(title, id, background) {
```

Comment:
Here is where we changed our parameters into an object because in the future we expected to encounter problems with local storage with the way it was set up before this change.